scene/2d/path_2d.cpp

```diff
@@ -315,8 +315,8 @@ void PathFollow2D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_rotates", "enable"), &PathFollow2D::set_rotates);
 	ClassDB::bind_method(D_METHOD("is_rotating"), &PathFollow2D::is_rotating);
 
-	ClassDB::bind_method(D_METHOD("set_cubic_interpolation", "enable"), &PathFollow2D::set_cubic_interpolation);
-	ClassDB::bind_method(D_METHOD("get_cubic_interpolation"), &PathFollow2D::get_cubic_interpolation);
+	ClassDB::bind_method(D_METHOD("set_cubic_interpolation", "enabled"), &PathFollow2D::set_cubic_interpolation);
+	ClassDB::bind_method(D_METHOD("is_cubic_interpolation"), &PathFollow2D::is_cubic_interpolation);
```

Comment:
Similar thoughts here, this seems unrelated, I would split into multiple PRs.

These renames also break compatibility, so it can't be done for 4.1. If you want to see these renames in 4.0 I suggest you make a PR with only these renames which can be merged in advance of the other changes. Otherwise this will have to wait for 5.0.

Reply:
Separated as #72842 and #72843.